src/lib/select/README.md

```diff
@@ -1,3 +1,68 @@
-## Work in progress!
+# md-select
 
-The select is still a work in progress, so most features have not been implemented. Not ready for use!
\ No newline at end of file
+`<md-select>` is a form control for selecting a value from a set of options, similar to the native
```

Comment:
FYI, I plan on wiping out all of the README files soon (and just pointing to the docs site)